src/core/jvm/insn/PaddedInstruction.js

```diff
@@ -0,0 +1,25 @@
+const AbstractInstruction = require('./AbstractInstruction');
+
+export default class PaddedInstruction extends AbstractInstruction {
+
+  constructor(idx, opcode) {
+    super(idx, opcode);
+  }
+
+  padding(offset) {
+    return super.size + this.paddingBytes(offset + 1);
+  }
+
+  paddingBytes(bytesCount) {
+    let bytesToPad = (4 - bytesCount % 4);
+    return (bytesToPad == 4 ? 0 : bytesToPad);
+  }
+
+  read(buffer) {
+    super.read(buffer);
+    let bytesToRead = paddingBytes(buffer.pos);
```

Comment:
`#paddingBytes` is undefined in this scope. you need to reference `this#paddingBytes()`